src/types.ts

```diff
@@ -93,6 +98,11 @@ export interface OptionsFormatters {
    */
   prettierOptions?: VendoredPrettierOptions
 
+  /**
+   * Custom configs let you have different configuration for certain file extensions and specific files.
+   */
+  prettierConfigs?: (options: PrettierConfigsOptions) => TypedFlatConfigItem[]
```

Comment:
I am not sure about the return of `TypedFlatConfigItem[]` convention, which supposedly to be the `append` or `prepend` operation of the main config. Also it feels a bit too specific here.

Reply:
Any suggestions to optimize it?

Reply:
I don't understand what you want - for appending the config you and use `.append` etc, I guess you'd more like to read the resolved options?

Reply:
I hope to allow users to use prettier intead of stylistic to format `GLOB_SRC`, but I can not get the internal runtime prettier options easily to keey the consistent format style.

Reply:
Also it will be easily to format others files by prettier if necessary.